.github/dependabot.yml

```diff
@@ -5,6 +5,11 @@ updates:
   schedule:
     interval: "daily" 
   target-branch: "main" 
+- package-ecosystem: "github-actions"
+  directory: "/.github/workflows"
```

Comment:
```suggestion
  directory: "/"
```

Reply:
I suggest here to check the whole dir including `/.github/workflows`, `/.github` and so on, all actions should be updated.